website/content/docs/drivers/qemu.mdx

```diff
@@ -61,6 +61,13 @@ The `qemu` driver supports the following configuration in the job spec:
   [alloc_dir](/docs/configuration/client#alloc_dir)
   paths.) This feature is currently not supported on Windows.
 
+- `guest_agent` `(bool: false)` - Enable the [QEMU Guest 
+  Agent](https://wiki.qemu.org/Features/GuestAgent) for this virtual machine.  This 
+  will add the necessary virtual hardware and create a `qemu-guest-agent.sock`
+  file in the task's working directory for interacting with the agent, which
+  must be running in the VM to work.  This feature is currently not supported
+  on Windows.
```

Comment:
The text here is slightly ambiguous because "agent" is a Nomad term as well, and we're only passing the socket flag. I think what you're saying here is that the QEMU Guest Agent has to be running in the guest virtual machine, right? Maybe this should read something like:
```suggestion
- `guest_agent` `(bool: false)` - Enable support for the [QEMU Guest 
  Agent](https://wiki.qemu.org/Features/GuestAgent) for this virtual machine. This 
  will add the necessary virtual hardware and create a `qemu-guest-agent.sock`
  file in the task's working directory for interacting with the agent. The QEMU Guest
  Agent must be running in the guest VM.  This feature is currently not supported
  on Windows.
```